contrib/spotifyd.service

```diff
@@ -7,7 +7,10 @@ Wants=network-online.target
 After=network-online.target
 
 [Service]
-ExecStart=/usr/bin/spotifyd --no-daemon
+User=spotify
+Group=spotify
```

Comment:
These don't match the user created in README `spoty`

Reply:
maybe spotify or spotifyd would be a better name?

Reply:
I have seen your comments just a moment ago . I will recreate this pull request and correct it straight away